tests/app/main/views/test_service_updates.py

```diff
@@ -302,33 +258,24 @@ def test_should_show_updates_if_valid_search(self, data_api_client):
 
         data_api_client.find_audit_events.return_value = audit_event
         response = self.client.get('/admin/service-updates?audit_date=2010-01-01')  # noqa
-        self.assertEquals(200, response.status_code)
-
-        self.assertIn(
-            self._replace_whitespace(
-               '<td class="summary-item-field-first"><span>Clouded Networks</span></td>'),  # noqa
-            self._replace_whitespace(response.get_data(as_text=True))
-        )
-        self.assertIn(
-            self._replace_whitespace(
-               '<td class="summary-item-field"><span>09:49:22<br/>17 June</span></td>'),  # noqa
-            self._replace_whitespace(response.get_data(as_text=True))
-        )
-        self.assertIn(
-            self._replace_whitespace(
-               '<td class="summary-item-field-with-action"><span><a href="/admin/services/compare/...">View changes</a></span></td>'),  # noqa
-            self._replace_whitespace(response.get_data(as_text=True))
-        )
-        self.assertIn(
-            self._replace_whitespace(
-               '<form action="/admin/service-updates/25/acknowledge" method="post">'),  # noqa
-            self._replace_whitespace(response.get_data(as_text=True))
-        )
-        self.assertIn(
-            self._replace_whitespace(
-               '<input name="audit_date" type="hidden" value="2010-01-01">'),  # noqa
-            self._replace_whitespace(response.get_data(as_text=True))
-        )
+        assert response.status_code == 200
+
+        assert self._replace_whitespace(
+            '<td class="summary-item-field-first"><span>Clouded Networks</span></td>'
+        ) in self._replace_whitespace(response.get_data(as_text=True))
+        assert self._replace_whitespace(
```

Comment:
I'd be tempted to add blank newlines between each of these multi-line asserts to make it a bit easier to scan (the brackets previously broke it up a bit).